tests/testing_utils.py

```diff
@@ -50,6 +51,12 @@ def run_command_with_input(command_list, stdin_input, cwd=None, env=None, timeou
         raise
 
 
+def run_command_with_inputs(command_list: List[str], inputs: List[str], timeout=TIMEOUT) -> CommandResult:
```

Comment:
should the arguments here match the full argument list of `run_command_with_input`? (missing `cwd`)